experimentation/src/main/java/com/automattic/android/experimentation/ExPlat.kt

```diff
@@ -62,9 +67,11 @@ class ExPlat(
     }
 
     fun clear() {
-        appLogWrapper.d(T.API, "ExPlat: clearing cached assignments and active variations")
+        logger.d("ExPlat: clearing cached assignments and active variations")
         activeVariations.clear()
-        experimentStore.clearCachedAssignments()
+        coroutineScope.launch {
```

Comment:
Minor (🔍): Consider making this a one-liner like you did with the rest (including the `runBlocking` block).